config/prometheus/gateway_plugin.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
seems there're pod annotation service discovery above, Do we still need `ServiceMonitor`?

Reply:
changes moved to https://github.com/vllm-project/aibrix/pull/1211
will move your comment there and discuss / address